const.env

```diff
@@ -0,0 +1 @@
+export API_KEY = '7ae9a13ef0f608cf1204686516291629569083a7'
```

Comment:
don't commit keys/ username / pw etc...

Reply:
+ don't commit env file for security purposes. create new const.envdummy & add above details with dummy data. Explain how to use this file as a comment